mavros_extras/mavros_plugins.xml

```diff
@@ -26,6 +26,9 @@
 	<class name="rangefinder" type="mavros::extra_plugins::RangefinderPlugin" base_class_type="mavros::plugin::PluginBase">
 		<description>Publish RANGEFINDER message data from FCU sensors in companion computer.</description>
 	</class>
+	<class name="rpm" type="mavros::extra_plugins::RPMPlugin" base_class_type="mavros::plugin::PluginBase">
+		<description>Publish RPM message data from FCU sensors in companion computer.</description>
```

Comment:
Please put an annotation referring this is a Ardupilot specific msg: Example: `Publish RPM message data from FCU sensors in companion computer. (Ardupilot only)`